README.md

```diff
@@ -122,6 +122,7 @@ Keep-Alive: timeout=5
 | trustProxy      | `TRUST_PROXY `            | `false`       | no       | By enabling the trustProxy option, Unleash Proxy will have knowledge that it's sitting behind a proxy and that the X-Forwarded-* header fields may be trusted, which otherwise may be easily spoofed. The proxy will automatically enrich the ip address in the Unleash Context. Can either be `true/false` (Trust all proxies), trust only given IP/CIDR (e.g. `'127.0.0.1'`) as a `string`. May be a list of comma separated values (e.g. `'127.0.0.1,192.168.1.1/24'` | 
 | namePrefix        | `UNLEASH_NAME_PREFIX`              | undefined        | no       | Used to filter features by using prefix when requesting backend values. | 
 | tags        | `UNLEASH_TAGS`              | undefined        | no       | Used to filter features by using tags set for features. Format should be `tagName:tagValue,tagName2:tagValue2` | 
+| proxyHeaderName        | `PROXY_HEADER_NAME`              | "authorization"        | no       | Used to specify under which header will be passed proxy secret | 
```

Comment:
Could we name it `proxySecretHeaderName` just to make it super clear that it is connected to the `proxySecrets` and nothing else.

Reply:
aligned + added test! Thank U ;)